tests/integration/test_integration_token_end_to_end.py

```diff
@@ -173,6 +174,15 @@ def test_nominal_case_of_an_api_user(client: TestClient, admin_access_token: str
     res = client.post(f"/v1/studies/{variant_id}/commands", headers=bot_headers, json=commands)
     assert res.status_code == 200
 
+    # check if the author's name and date of update are retrieved with commands created by a bot
+    commands_res = client.get(f"/v1/studies/{variant_id}/commands", headers=bot_headers)
+    assert commands_res.status_code == 200
+    assert commands_res.json()
+
+    for command in commands_res.json():
+        assert command.get("user_name") in ["admin", "admin_bot"]
```

Comment:
Could you check strictly that each command has the correct user name ?
Also minor but I prefer to assert "updated_at" is inside the command than doing a get as I find it less readable but not really important